### ISSUE TITLE
Alt Text For Sort Icon In Table Sorting Header

### ISSUE THREAD

**Issue by ajay1208** [angular/components#17707]
#### Reproduction
In Angular material table, while we do sorting we get sort icons.  The sort icons don't have corresponding alt text. This fails accessibility.
 
 
#### Expected Behavior
Alt Text are not there for sort icon.

#### Actual Behavior

Sort icon should ave Alt Text

**Comment by crisbeto** [comment]
We aren't using an `img` tag for the icon so an `alt` won't do anything. Closing since the sort header already has an `aria-label` for accessibility purposes.
